libsolidity/analysis/NameAndTypeResolver.cpp

```diff
@@ -102,7 +102,12 @@ bool NameAndTypeResolver::performImports(SourceUnit& _sourceUnit, map<string, So
 					else
 						for (Declaration const* declaration: declarations)
 							if (!DeclarationRegistrationHelper::registerDeclaration(
-								target, *declaration, alias.alias.get(), &alias.location, false, m_errorReporter
+								target,
+								*declaration,
+								alias.alias ? alias.alias.get() : &alias.symbol->name(),
```

Comment:
So the problem was that if you have `import {X} from "y.sol"`, then `alias.alias` would be `nullptr` and it would use the "native name" of the symbol? So this only happens if you have two imports and the name is changed in the first import, right?

Reply:
Can you maybe clarify this a bit more in the changelog, i.e. say that it fixes `import {name} from "y.sol";` if `name` is not the "native name" of the symbol?